### PR TITLE
feat(settings-compat-v7): added text decoration in breadcrumb component

### PR DESCRIPTION
**Pull request by ddmarin94**
Added text decoration underline in breadcrumb component setted by default no none in ord
er to be available set in the theme of each site whatever they want, underline, overline,
 etc.

Also fixed a lint error convert 0px into 0

### CODE REVIEW

File: src/settings-compat-v7/_components.scss [hunk 1]
```diff
@@ -131,6 +131,7 @@ $c-breadcrumb-link: $c-primary !default;
 $c-breadcrumb: $c-text-base !default;
 $m-breadcrumb-items: $m-h-small !default;
 $size-breadcrumb-icon: 18px !default;
+$td-breadcrumb-link-hover: none !default;
```
Comment (jcger) [inline src/settings-compat-v7/_components.scss]: Would it be much asking if you could be so kind to move all these variables into the component and remove breadcrumb's settings-compat-v7 dependency?

Comment (ddmarin94) [reply src/settings-compat-v7/_components.scss]: Sure no problem, I will move all these breadcrumb variables inside components/_breadcrumb.scss  :)

Comment (jcger) [reply src/settings-compat-v7/_components.scss]: Thx! but you should move them to https://github.com/SUI-Components/sui-components/blob/master/components/molecule/breadcrumb/src/index.scss and remove the first line

Comment (ddmarin94) [reply src/settings-compat-v7/_components.scss]: @jcger removed, take a look here please --> https://github.com/SUI-Components/sui-components/pull/546/files

**Pull request merged**